app/controllers/api/content_controller.rb

```diff
@@ -0,0 +1,24 @@
+class Api::ContentController < Api::BaseController
+  before_action :validate_params!
+  def index
+    @content = Reports::Content.retrieve(from: params[:from], to: params[:to],
```

Comment:
Would it be possible to have all the parameters with the same layout?

Something like:

```ruby
@content = Reports::Content.retrieve(
  from: params[:from],
  to: params[:to],
  organisation: params[:organisation],
)
```

or 

```ruby
@content = Reports::Content.retrieve(from: params[:from], to: params[:to], organisation: params[:organisation]
```